sites/skeleton.dev/src/content/docs/components/progress-circular/svelte.mdx

```diff
@@ -0,0 +1,63 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+---
+
+export const components = componentSet;
+
+import Default from '@examples/components/progress-circular/svelte/default.svelte';
+import DefaultRaw from '@examples/components/progress-circular/svelte/default.svelte?raw';
+
+import Indeterminate from '@examples/components/progress-circular/svelte/indeterminate.svelte';
+import IndeterminateRaw from '@examples/components/progress-circular/svelte/indeterminate.svelte?raw';
+
+import Format from '@examples/components/progress-circular/svelte/format.svelte';
+import FormatRaw from '@examples/components/progress-circular/svelte/format.svelte?raw';
+
+import CustomFormat from '@examples/components/progress-circular/svelte/custom-format.svelte';
+import CustomFormatRaw from '@examples/components/progress-circular/svelte/custom-format.svelte?raw';
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<Default client:visible />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={DefaultRaw} lang="svelte" />
+	</Fragment>
+</Preview>
+
+## Indeterminate
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<Indeterminate client:visible />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={IndeterminateRaw} lang="svelte" />
+	</Fragment>
+</Preview>
+
+## Format
+
```

Comment:
We need to explain format. It's got at least 2-3 options. So let's at least "tell" if not "show" as well. Nothing else on this docs explain this. And we should link to the browser `Intl` API - which I included in my original comment for this    demo.